easybuild/easyblocks/l/lammps.py

```diff
@@ -118,6 +128,8 @@
     'zen2': 'ZEN2',
     'zen3': 'ZEN3',
     'power9le': 'POWER9',
+    'neoverse_n1': 'ARMV81',
+    'neoverse_v1': 'ARMV81',
```

Comment:
add `a64fx`:

```
   'a64fx': 'A64FX',
```

supported since LAMMPS version `stable_2Aug2023`, see https://github.com/lammps/lammps/commit/60864e38d1d5f34db0ff379bf71eab0da0ca1ed0